Doc/howto/index.rst

```diff
@@ -35,3 +33,31 @@ Currently, the HOWTOs are:
    timerfd.rst
    mro.rst
 
+General:
+
+* :ref:`annotations-howto`
+* :ref:`argparse-tutorial`
+* :ref:`descriptorhowto`
+* :ref:`enum-howto`
+* :ref:`functional-howto`
+* :ref:`ipaddress-howto`
+* :ref:`logging-howto`
+* :ref:`logging-cookbook`
+* :ref:`regex-howto`
+* :ref:`sortinghowto`
+* :ref:`unicode-howto`
+* :ref:`urllib-howto`
+
+Advanced development:
+
+* :ref:`curses-howto`
+* :ref:`isolating-extensions-howto`
+* :ref:`python_2.3_mro`
+* :ref:`socket-howto`
+* :ref:`timerfd-howto`
```

Comment:
“Porting Extension Modules to Python 3” is a stub nowadays, but I don't think it should be removed in a reorganization PR.
```suggestion
* :ref:`timerfd-howto`
* :ref:`cporting-howto`
```